Products/PloneBooking/skins/plonebooking_javascripts/plonebooking.js

```diff
@@ -783,17 +808,28 @@ Booking.submit = function (e) {
     xhReq.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
     xhReq.onreadystatechange = function() {
         if (xhReq.readyState != 4) return null;
-        if (xhReq.responseText != 'True') {
-            response = xhReq.responseText;
-            if (response.split('\n').length > 1)
+             // alert(isHTML(xhReq.responseText));  // should return true
+             // alert(isHTML("Phone number")); // should return false
+             if (isHTML(xhReq.responseText) == true) {
+                 var retvalue = $(xhReq.responseText).text().trim();
+             } else { 
+                 var retvalue = xhReq.responseText
+             }
+             if (retvalue != 'True') {
+                //response = xhReq.responseText.substring(retvalue.indexOf('\n') + 1); // we jump over the first line which contains doctype like: <!DOCTYPE html>
+                if (retvalue.split('\n').length > 1) 
+       //Plone 4 if (xhReq.responseText != 'True') {
+       //Plone 4     response = xhReq.responseText;
+       //Plone 4     if (response.split('\n').length > 1)              
```

Comment:
Do you need to have code that behaves differently here for Plone 4 vs 5? If so, is there a better way to do this? Leaving comments here makes it ambiguous.